content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Secure Supply Chain: Verifying image signatures"
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+After these last releases Kubewarden now has support for verifying the integrity and authenticity of artefacts within
+Kubewarden using the Sigstore project. In this post, we shall focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
```

Comment:
Personal taste, feels more informal and doesn't put pressure in understanding how sigstore works to keep reading this blog post

```suggestion
To learn more about how Sigstore works, take a look at our
```